packages/app/src/app/components/CreateNewSandbox/CreateSandbox/SearchBox/elements.ts

```diff
@@ -42,24 +46,3 @@ export const SearchElement = styled.input`
     color: #757575;
   }
 `;
-
-export const InputWrapper = styled.div`
```

Comment:
I think this used to add the press slash to focus indicator?

Reply:
right, will add that back, I forgot about it